en/02_Developer_Guides/15_Customising_the_Admin_Interface/01_ModelAdmin.md

```diff
@@ -167,7 +167,7 @@ $tabLink = $admin->getLinkForModelTab('product-category');
 > [getModelTabForModelClass()](api:SilverStripe\Admin\ModelAdmin::getModelTabForModelClass()) method
 > for your `ModelAdmin` subclass.
 
-You can also use the new [CMSEditLinkExtension](api:SilverStripe\Admin\CMSEditLinkExtension) to provide a `CMSEditLink()` method on the record - see [Managing Records](../model/managing_records#getting-an-edit-link).
+You can also use the new [CMSEditLinkExtension](api:SilverStripe\Admin\CMSEditLinkExtension) to update the value of the `CMSEditLink()` method on the record - see [Managing Records](../model/managing_records#getting-an-edit-link).
```

Comment:
Should this be more along the links of "you can use the updateCMSEditLink() extension hook to update the value"?

Also CMSEditLinkExtension is no longer "new" :-)

Reply:
That could be stated separately if you want it to be. But that extension exists to do that for you and calling out here is intentional.